packages/tenant-process/test/utils.ts

```diff
@@ -193,7 +194,7 @@ export const addOneTenant = async (
   tenants: TenantCollection
 ): Promise<void> => {
   await writeTenantInEventstore(tenant, postgresDB);
-  await writeInReadmodel(tenant, tenants);
+  await writeInReadmodel(toReadModelTenant(tenant), tenants);
```

Comment:
Could you apply the change also to other test utils where we use `writeInReadmodel(tenant, tenants);` in other services?
TS compiler does not complain because we don't include test files in the build, but VSCode will show you the TS errors when you open the files